dask/dataframe/shuffle.py

```diff
@@ -179,22 +186,18 @@ def sort_values(
         repartition = False
 
     sort_by_col = df[by[0]]
-
-    if not isinstance(ascending, bool):
-        # support [True] as input
-        if (
-            isinstance(ascending, list)
-            and len(ascending) == 1
-            and isinstance(ascending[0], bool)
-        ):
-            ascending = ascending[0]
-        else:
-            raise NotImplementedError(
-                f"Dask currently only supports a single boolean for ascending. You passed {str(ascending)}"
-            )
-
+    divisions_ascending = ascending
+    if divisions_ascending and not isinstance(divisions_ascending, bool):
+        divisions_ascending = divisions_ascending[0]
+    assert divisions_ascending is None or isinstance(divisions_ascending, bool)
```

Comment:
What happens if I put `ascending=[None, False]`?

Reply:
then `divisions_ascending` will be `None` which is the default, i.e. the divisions sorting (and the column) will use the default.

At least I don't see a reason why that shouldn't work. I can add a test case

Reply:
> then `divisions_ascending` will be `None` which is the default, i.e. the divisions sorting (and the column) will use the default.

Makes sense, though I guess it'll blow up in `pandas` later in the process.

Reply:
yeah, pandas goes 💥 

```python-traceback
./../miniforge3/envs/dask-distributed/lib/python3.10/site-packages/pandas/core/series.py:3721: in sort_values
    ascending = validate_ascending(ascending)
../../miniforge3/envs/dask-distributed/lib/python3.10/site-packages/pandas/util/_validators.py:360: in validate_ascending
    return validate_bool_kwarg(ascending, "ascending", **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

value = None, arg_name = 'ascending', none_allowed = False, int_allowed = True

    def validate_bool_kwarg(
        value: BoolishNoneT,
        arg_name: str,
        none_allowed: bool = True,
        int_allowed: bool = False,
    ) -> BoolishNoneT:
        """
        Ensure that argument passed in arg_name can be interpreted as boolean.

        Parameters
        ----------
        value : bool
            Value to be validated.
        arg_name : str
            Name of the argument. To be reflected in the error message.
        none_allowed : bool, default True
            Whether to consider None to be a valid boolean.
        int_allowed : bool, default False
            Whether to consider integer value to be a valid boolean.

        Returns
        -------
        value
            The same value as input.

        Raises
        ------
        ValueError
            If the value is not a valid boolean.
        """
        good_value = is_bool(value)
        if none_allowed:
            good_value = good_value or (value is None)

        if int_allowed:
            good_value = good_value or isinstance(value, int)

        if not good_value:
>           raise ValueError(
                f'For argument "{arg_name}" expected type bool, received '
                f"type {type(value).__name__}."
            )
E           ValueError: For argument "ascending" expected type bool, received type NoneType.

../../miniforge3/envs/dask-distributed/lib/python3.10/site-packages/pandas/util/_validators.py:264: ValueError
```

Reply:
nit: It would be nicer if we caught that earlier with a nice error message, but that's definitely non-blocking.